charts/perses/values.yaml

```diff
@@ -101,10 +101,11 @@ config:
 
   # -- Schemas paths
   schemas:
-    panels_path: "/etc/perses/schemas/panels"
-    queries_path: "/etc/perses/schemas/queries"
-    datasources_path: "/etc/perses/schemas/datasources"
-    variables_path: "/etc/perses/schemas/variables"
+    panels_path: "/etc/perses/cue/schemas/panels"
+    queries_path: "/etc/perses/cue/schemas/queries"
+    datasources_path: "/etc/perses/cue/schemas/datasources"
+    variables_path: "/etc/perses/cue/schemas/variables"
+    interval: "6h"
```

Comment:
@Nexucis I have seen here different default values, the values.schema.json has it referenced as 5m, the docs 1h. What is a reasonable consistent default here?

Reply:
The initial idea was 5m. That was in case the watching file is failing to update the schemas loaded.

Also after many issues we ran into with cuelang, keeping `5m` is a good value because it helps to trash the cuelang context that is leaking. So it mitigates this memory leak